design/proposals/dbt-delete-merge-schema-branch.md

```diff
@@ -0,0 +1,30 @@
+# `lakectl dbt delete-branch-schema` & `lakectl dbt merge-branch-schema`
+
+## Description
+When using dbt alongside lakeFS, the first thing we would want to do is to create a branch to work on. That branch is created with a dbt schema associated with it (it might have the same name of the branch [by default] or a custom one) that is basically a copy of the schema that was associated with the branch which we branched out from but with different name and location.   
+After running dbt over that branch and conducting tests and validations we would want to either: 
+- delete the branch and the schema associated with it
+- merge them into master/main/production/other branch
+- both.
+
+`lakectl dbt delete-branch-schema` will allow the deletion of a branch and the schema associated with it, or just the schema.  
+`lakectl dbt merge-branch-schema` will allow the merging of a branch to a given branch  and the schema associated with it. 
+### Why do we need it?
+The users that use the dbt command are, well, dbt users. They will interact with the `lakectl` tool in the context of the dbt command (and some basic commands such as `commit` and `merge`).
+Adding the `delete-branch-schema` and `merge-branch-schema` subcommands will make it easier for them to integrate lakeFS into their framework.
+
+## `delete-branch-schema` Usage
+1. `lakectl dbt delete-branch-schema --branch <branch name> --schema <schema name>`: Delete branch and schema with the given names.
```

Comment:
a general question - 
Is there a way for users to get the schema associated with a branch via lakectl? I mean, as a user, how do I know the name of the schema I'm messing with?